src/components/FlightRow/FlightRow.tsx

```diff
@@ -1 +1,26 @@
 // row component for the table of flights should go here
+
+import { Flight } from "../../App"
+
+interface FlightRowProps{
+    flight: Flight
+}
+
+
+const FlightRow = ({flight} : FlightRowProps) => {
+    return(
+        <div className="ROW">
+            {flight.flightNumber + " " + " "}
+            
+            {flight.arrivalTime + "   " + "  "}
+            {" " + flight.departureTime + "     " + " " + " "}
+            {flight.flightDuration + "    " + " "}
+            {flight.origin + "     "}
+            {flight.destination}
+
+        </div>
```

Comment:
We can use table elements to make this more table-ish

The `<tr>` elements a "table row" and the `<td>` elements denotes "table data".

I've even added some padding to give out items the spacing they need!

```suggestion
  <tr>
      <td style={{ padding: "10px" }}>
          {flight.flightNumber}
      </td>
       <td style={{ padding: "10px" }}>
           {flight.arrivalTime}
       </td>
       <td style={{ padding: "10px" }}>
           {flight.departureTime}
       </td>
       <td style={{ padding: "10px" }}>
           {flight.flightDuration}
       </td>
       <td style={{ padding: "10px" }}>
           {flight.origin}
       </td>
       <td style={{ padding: "10px" }}>
           {flight.destination}
       </td>
  </tr>
```